### DISCUSSION TITLE
1.8

### DISCUSSION THREAD

**Discussion by kirb** [LegacyUpdate/LegacyUpdate#184] [Announcements]
* Legacy Update is now signed, so you won’t see any more scary warnings about “program from an unknown publisher” or similar.  
  **Code signing certificates are expensive - this cost me $340 USD. I was only able to do this thanks to those who generously support the project on [GitHub Sponsors](https://github.com/sponsors/kirb) and [Patreon](https://www.patreon.com/adamdemasi).**
* Legacy Update can now be directly accessed from a regular, non-administrator instance of Internet Explorer on Windows Vista and later. This also means you can use Legacy Update via Internet Explorer Mode in Microsoft Edge.
* Provides better messaging when Internet Explorer has been uninstalled (such as in “E” editions of Windows).
* Fixes Legacy Update setup unnecessarily redownloading updates that are already installed.
* Fixes an error when attempting to uninstall Legacy Update from the Control Panel.
* Hopefully fixes the “Enable Microsoft Update” option in Legacy Update setup not working on Windows 7.

<hr /><em>This discussion was created from the release <a href='https://github.com/kirb/LegacyUpdate/releases/tag/v1.8.0.0'>1.8</a>.</em>

**Comment by Glinckey** [comment]
![image](https://github.com/kirb/LegacyUpdate/assets/105798654/64a329ed-35b0-4381-8b34-923be2286f1e)
